wiki/Beatmap_Spotlights/Seasons/2020_Autumn/zh.md

```diff
@@ -29,12 +29,12 @@ tags:
 
 ### 每周多人游戏大厅
 
-目前，加入每周多人游戏大厅需要使用 [osu!(lazer)](https://github.com/ppy/osu/releases)。
+目前，加入每周多人游戏大厅需要使用 [osu!lazer](https://github.com/ppy/osu/releases)。
 
 1. 每个大厅将持续开放一周
 2. 图池现可包含 Mod
    - 带 Mod 的谱面与无 Mod 的谱面相同，目的是为了给更有经验的玩家带来挑战性。
-   - 由于 osu!(lazer) 的限制，osu!taiko 在本赛季为 NoMod。
+   - 由于 osu!lazer 的限制，osu!taiko 在本赛季为 NoMod。
```

Comment:
we're not using "osu!lazer" anymore. either use "osu!(lazer)" for now or refer to https://github.com/ppy/osu-wiki/issues/6590 to reword it

Reply:
Thanks for the issue refer, that's news for me.
But I actually did it for a reason, in order to consider the formatting coherence of the articles, according to the criterion (e.g. Chinese copywriting guidelines), fullwidth Chinese symbols should be used unless a full English whole sentence is encountered, it would be weird to apply it here, so I chose the former one.
But now that there are the new criteria, I will change it back. Thanks.

Reply:
`osu!（lazer）` is ok too

Reply:
i prefer `osu!(lazer)`, cuz its should be a proper noun, also the `osu!（lazer）` is looks weird in hyperlinks: `目前，加入每周多人游戏大厅需要使用 [osu!（lazer）](https://github.com/ppy/osu/releases)。`
and I created a single pr to format the symbols in all of the Chinese articles last year, #3848. But after that, the Chinese translation pr quality is pretty high. There is not much need. I will pay attention to it.